src/DockerFileGenerator.php

```diff
@@ -32,7 +31,8 @@ final class DockerFileGenerator
     private const FILE_TEMPLATE = <<<'Dockerfile'
         FROM php:__BASE_PHP_IMAGE_TOKEN__
 
-        RUN $(php -r '$extensionInstalled = array_map("strtolower", \get_loaded_extensions(false));$requiredExtensions = __PHP_EXTENSIONS_TOKEN__;$extensionsToInstall = array_diff($requiredExtensions, $extensionInstalled);if ([] !== $extensionsToInstall) {echo \sprintf("docker-php-ext-install %s", implode(" ", $extensionsToInstall));}echo "echo \"No extensions\"";')
+        COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+        RUN /usr/bin/install-php-extensions __REQUIRED_EXTENSIONS__
```

Comment:
what happens when there is no extension? is `RUN /usr/bin/install-php-extensions` alone working fine?

Reply:
Yes, just confirmed this.